packages/vscode-extension/src/debugging/DebugSession.ts

```diff
@@ -168,6 +172,23 @@ export class DebugSessionImpl implements DebugSession, Disposable {
     );
   }
 
+  public async findOriginalPosition(sourceInfo: SourceInfo): Promise<SourceInfo> {
+    try {
+      const response = await this.jsDebugSession?.customRequest(
```

Comment:
We should have a guard for `jsDebugSession`

```suggestion
    if (this.jsDebugSession === undefined) {
      throw new Error("Could not get original source position, JS Debugger is not connected.");
    }
    try {
      const response = await this.jsDebugSession.customRequest(
```